examples/trt-external-payload-cluster-density.yaml

```diff
@@ -12,6 +12,7 @@ tests :
       benchmark.keyword: cluster-density-v2
       ocpVersion: {{ version }}
       networkType: OVNKubernetes
+      jobType: periodic OR pull
```

Comment:
I think for the sake of our nightly release stream configs, I only want to see 'periodic' runs that have been triggered from a nightly. For the three `trt-external-payload*` configs in this PR.

If we include 'pull' tests, I think we open the door to report a 'fail' on a nightly based on a PR run that wasn't actually merged.

Reply:
I see it updated. thanks!